File: templates/lang/android_app_dagger/src_files/test/ut/BuildMockspresso.java
package {{ .Group }};

import com.episode6.hackit.mockspresso.quick.QuickMockspresso;

import {{ .Group }}.app.MockspressoTestApp;

import static com.episode6.hackit.mockspresso.quick.QuickBuildMockspresso.with;

public class BuildMockspresso {
  public static QuickMockspresso.Builder withDefaults() {
    return with()
        .injector().dagger()
        .mocker().mockito();
  }

  public static QuickMockspresso.Builder forRobolectric() {
    return MockspressoTestApp.get().buildUpon();
  }
}

File: templates/lang/kotlin_android_app_dagger/src_files/test/ut/BuildMockspresso.kt
package {{ .Group }}

import {{ .Group }}.app.MockspressoTestApp
import com.episode6.hackit.mockspresso.quick.QuickMockspresso
import com.episode6.hackit.mockspresso.quick.BuildQuickMockspresso

object BuildMockspresso {
  fun withDefaults(): QuickMockspresso.Builder {
    return BuildQuickMockspresso.with()
        .injector().dagger()
        .mocker().mockito()
  }

  fun forRobolectric(): QuickMockspresso.Builder {
    return MockspressoTestApp.get().buildUpon()
  }
}
